Formula/glib.rb

```diff
@@ -80,6 +87,18 @@ def install
     #       directories that only contain empty directories.
     prefix.find.select(&:directory?).reverse_each { |d| d.rmdir if d.empty? }
 
+    # Some Python executables are removed and ship with `glib-utils`.
+    # Fix the pkg-config variable paths.
+    glib_utils_bin = Formula["glib-utils"].opt_bin
+    removed_python_executables.each do |old_pyexec|
+      new_pyexec = old_pyexec.sub("${bindir}", glib_utils_bin)
+      (lib/"pkgconfig").glob("**/*.pc").each do |f|
+        inreplace f, old_pyexec, new_pyexec
+          rescue
+            next # string not found
```

Comment:
How to `inreplace` and not raise errors when no match is found?

Reply:
There shouldn't be an inreplace if there is nothing to replace

Reply:
> There shouldn't be an inreplace if there is nothing to replace

There are only two places we need to `inreplace`. Should we hardcode that rather than loop over all `.pc` files and potential replacements?

In `glib/lib/pkgconfig/gioi-2.0.pc`:

```diff
 prefix=/home/linuxbrew/.linuxbrew/Cellar/glib/2.72.3
 includedir=${prefix}/include
 libdir=${prefix}/lib

 datadir=${prefix}/share
 schemasdir=${datadir}/glib-2.0/schemas
 bindir=${prefix}/bin
 giomoduledir=${libdir}/gio/modules
 gio=${bindir}/gio
 gio_querymodules=${bindir}/gio-querymodules
 glib_compile_schemas=${bindir}/glib-compile-schemas
 glib_compile_resources=${bindir}/glib-compile-resources
 gdbus=${bindir}/gdbus
-gdbus_codegen=${bindir}/gdbus-codegen
 gresource=${bindir}/gresource
 gsettings=${bindir}/gsettings

 Name: GIO
 Description: glib I/O library
 Version: 2.72.3
 Requires: glib-2.0, gobject-2.0
 Requires.private: gmodule-no-export-2.0, zlib, mount >= 2.23
 Libs: -L${libdir} -lgio-2.0
 Libs.private: -ldl -lresolv
 Cflags: -I${includedir}
```

In `glib/lib/pkgconfig/glib-2.0.pc`:

```diff
 prefix=/home/linuxbrew/.linuxbrew/Cellar/glib/2.72.3
 includedir=${prefix}/include
 libdir=${prefix}/lib

 bindir=${prefix}/bin
-glib_genmarshal=${bindir}/glib-genmarshal
 gobject_query=${bindir}/gobject-query
-glib_mkenums=${bindir}/glib-mkenums

 Name: GLib
 Description: C Utility Library
 Version: 2.72.3
 Requires.private: libpcre >=  8.31
 Libs: -L${libdir} -lglib-2.0
 Libs.private: -pthread -lm
 Cflags: -I${includedir}/glib-2.0 -I${libdir}/glib-2.0/include
```

Reply:
Yes, explicitly defining which files to modify with Homebrew specific changes seems like the better option